### PR TITLE
Spike electron web contents view

### PR DESCRIPTION
**Pull request by priya-patel04**
# Description

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added `a11y-tests` label to run a11y audit tests if needed

## PCI review checklist

- [ ] I have documented a clear reason for, and description of, the change I am making.
- [ ] If applicable, I've documented a plan to revert these changes if they require more than reverting the pull request.
- [ ] If applicable, I've documented the impact of any changes to security controls.
  Examples of changes to security controls include using new access control methods, adding or removing logging pipelines, etc.

### CODE REVIEW

File: ui/desktop/app/components/session/tabs/index.js [hunk 1]
```diff
@@ -37,104 +35,59 @@ export default class SessionTerminalTabsComponent extends Component {
...
     window.onresize = () => {
       debouncedFit();
     };
-    xterm.onResize((size) => {
-      window.terminal.resize(size, this.id);
-    });
+  }
+
+  @action
+  openDetails() {
```
Comment (ZedLi) [inline ui/desktop/app/components/session/tabs/index.js]: Why do we want the terminal view destroyed when we navigate to the details? How would a user copy over brokered credentials?

File: ui/desktop/app/controllers/application.js
Comment (ZedLi) [inline ui/desktop/app/controllers/application.js]: Why are we doing any terminal management in the application controller?

File: ui/desktop/electron-app/config/web-contents-view.config.js
Comment (ZedLi) [inline ui/desktop/electron-app/config/web-contents-view.config.js]: I think it's OK if it works but I found `esbuild` to be much simpler to use compared to webpack especially if it's a straightforward one off

**Review by ZedLi — COMMENTED** [review]
Overall I think this is a great poc and I think is a very reasonable choice moving forward for better isolation of the terminal. I think we might have a few edge cases we need to properly test but I think this looks good!
